lib/Doctrine/ORM/Tools/Pagination/LimitSubqueryOutputWalker.php

```diff
@@ -39,17 +49,17 @@ class LimitSubqueryOutputWalker extends SqlWalker
     /**
      * @var \Doctrine\DBAL\Platforms\AbstractPlatform
      */
-    private $platform;
+    private $_platform;
```

Comment:
@stof - I actually ran into some really strange behavior with this. These properties were hiding the private ones from the parent class, but when I overrode walkSelectStatement, I was getting a fatal error saying that these private variables were undefined. It was really odd. I'll try backing the underscores out and see if I can reproduce.

Ideally these variables should just be protected on SqlWalker, but I didn't want to push changes too far out from LimitSubqueryOutputWalker.